src/runtime/server.ts

```diff
@@ -295,12 +336,17 @@ export function run() {
     }
   }
 
-  let app = createExpressApp();
+  var databases;
+  if (config.initJsonDB) {
+    databases = loadDatabases(config.initJsonDB);
+  }
+
+  let app = createExpressApp(databases);
```

Comment:
Rather than plumbing the list of databases throughout, let's attach it to the config. That way we can access it directly where it's needed. In the future if we want to have separate instances running in the same process for some reason we can give each instance its own config object.